### ISSUE TITLE
--search not working together with -u flag

### ISSUE THREAD

**Issue by Chubykalo** [twintproject/twint#973]
# Issue Template
Please use this template!

### Initial Check
> If the issue is a request please specify that it is a request in the title (Example: [REQUEST] more features). If this is a question regarding 'twint' please specify that it's a question in the title (Example: [QUESTION] What is x?). Please **only** submit issues related to 'twint'. Thanks.

>Make sure you've checked the following:

- [x] Python version is 3.6; --I have version 3.8.5
- [x] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [x] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran

FROM TERMINAL
twint -u realdonaldtrump --search "loser" 

FROM PYTHON

import twint

c = twint.Config()
c.Username = "santi_ABASCAL"
c.Search = "vox"
c.Since = "2020-02-01"

twint.run.Search(c)

### Description of Issue
the search flag does not seem to be working together with the -u flag. They work fine separately or with other flags. I have tested this with many different Twitter accounts.
When the command is run I get ''No more data. finished scraping!! is it though? because sometimes twitter lie.'' 
When using advanced search in Twitter, Mr.Trump naturally uses the word ''loser'' a lot, so there should be plenty of results.
Thank you for any help in advance.

EDIT: UPDATE. It seems the -s works with hashtags but not with the words found in the Twitter content. What am I doing wrong?
UPDATE2: the -s or search function works with the hashtag, but does not work otherwise. It will return tweets if I search ''#sampletext'' but it WILL NOT return tweets if I search ''sampletext''

### Environment Details
Using Ubuntu 20.04 from Terminal

**Comment by ishmaxs** [comment]
I too am getting the same issue here

**Comment by Chubykalo** [comment]
Thank you very much for the response. 

I will try this today and let you know. In the meantime I scraped ALL tweets from 321 desired accounts (185k) and used power query to filter :P Basically fishing by throwing bombs in the lake.

**Comment by himanshudabas** [comment]
> I scraped ALL tweets from 321 desired accounts (185k) and used power query to filter :P Basically fishing by throwing bombs in the lake.

Haha that's one way to do it.
But you should do what @ishmaxs suggested. Username and Search currently can't work together. It for that matter not many things work right now because twitter deprecated their older endpoints. So most of the functionality is broken right now. I implemented the search functionality.

Also I tried to fix some of the other things too, but my PR wasn't merged. Hope it gets merged soon so I can start fixing some more bugs and start adding some cool new features. ;)

**Comment by Chubykalo** [comment]
Hi everyone. Thank you very much for the help. With the advice of @ishmaxs it works. I'll leave the code I used here in case somebody wants to scrape from a list of twitter accounts on an excel column, which is what I did.

import twint
import pandas as pd

dataframe = pd.read_excel('filePATH)
twitter_list = dataframe['column name'].tolist()  

for twitter_handle in twitter_list:

    c = twint.Config()
    c.Lang='es'
    c.Search = 'from:@' + twitter_handle + 'string search filter'
    c.Since = "2020-01-01"
    c.Store_csv = True
    c.Output = "filename.csv"
    c.Count = True

    twint.run.Search(c)
